### PR TITLE
Revert "Allow CuPy 11"

### PR DESCRIPTION
**Pull request by galipremsagar**
Reverts rapidsai/cuxfilter#383

### CODE REVIEW

**Comment by galipremsagar** [comment]
xref: https://github.com/rapidsai/cudf/pull/11409

**Review by ajschmidt8 — COMMENTED** [review]
@galipremsagar, I think this repository was out-of-date with the rest of RAPIDS. can you update these references to `cupy >=7.8.0,<11.0.0a0`

File: conda/environments/cuxfilter_dev_cuda11.0.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ channels:
 - conda-forge
 - anaconda
 dependencies:
-- cupy>7.1.0,<12.0.0a0
+- cupy>7.1.0,<10.0.0a0
```
Comment (galipremsagar) [inline conda/environments/cuxfilter_dev_cuda11.0.yml:10]:
```suggestion
- cupy>=9.5.0,<11.0.0a0
```

File: conda/environments/cuxfilter_dev_cuda11.2.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ channels:
 - conda-forge
 - anaconda
 dependencies:
-- cupy>7.1.0,<12.0.0a0
+- cupy>7.1.0,<10.0.0a0
```
Comment (galipremsagar) [inline conda/environments/cuxfilter_dev_cuda11.2.yml:10]:
```suggestion
- cupy>=9.5.0,<11.0.0a0
```

File: conda/environments/cuxfilter_dev_cuda11.4.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ channels:
 - conda-forge
 - anaconda
 dependencies:
-- cupy>7.1.0,<12.0.0a0
+- cupy>7.1.0,<10.0.0a0
```
Comment (galipremsagar) [inline conda/environments/cuxfilter_dev_cuda11.4.yml:10]:
```suggestion
- cupy>=9.5.0,<11.0.0a0
```

File: conda/environments/cuxfilter_dev_cuda11.5.yml [hunk 4]
```diff
@@ -7,7 +7,7 @@ channels:
 - conda-forge
 - anaconda
 dependencies:
-- cupy>7.1.0,<12.0.0a0
+- cupy>7.1.0,<10.0.0a0
```
Comment (galipremsagar) [inline conda/environments/cuxfilter_dev_cuda11.5.yml:10]:
```suggestion
- cupy>=9.5.0,<11.0.0a0
```

**Comment by galipremsagar** [comment]
> @galipremsagar, I think this repository was out-of-date with the rest of RAPIDS. can you update these references to `cupy >=7.8.0,<11.0.0a0`

Done 👍 , also discussed offline and corrected the outdated min versions too.

**Review by ajschmidt8 — APPROVED** [review]

**Comment by ajschmidt8** [comment]
@gpucibot merge

**Pull request merged**